spec/factories/permissions.rb

```diff
@@ -19,4 +19,8 @@
     name 'Administrator'
   end
 
+  factory :system_permission, parent: :permission do
+    name 'System'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.